src/main/java/org/synyx/urlaubsverwaltung/application/service/ApplicationRecipientService.java

```diff
@@ -84,6 +86,31 @@
         return concat(bosses, relevantBosses, responsibleDepartmentHeads);
     }
 
+    /**
+     * Returns the relevant recipients for a given person.
+     * <p>
+     * If the person is in a department than return
+     * - the persons with the role department head for this user
+     * else return
+     * - all persons with the notification type {@code NOTIFICATION_OFFICE}
+     *
+     * @param application that has been rejected
+     * @return all relevant recipients to inform
+     */
+    List<Person> getRelevantRecipients(Application application) {
+        final List<Person> recipients = new ArrayList<>();
+
+        final List<Person> responsibleDepartmentHeads = getResponsibleDepartmentHeads(application.getPerson());
+        if (!responsibleDepartmentHeads.isEmpty()) {
```

Comment:
Office does not get any notification at all if person is in a department with departmentHead. But departmentHead is not allowed to do the cancelation

Reply:
hm I do not get it. Why is the departmentHead not allowed to do this? If we take a look at

```
        <sec:authorize access="hasAnyAuthority('DEPARTMENT_HEAD', 'SECOND_STAGE_AUTHORITY', 'BOSS')">
            <jsp:include page="actions/allow_form.jsp"/>
            <jsp:include page="actions/reject_form.jsp"/>
            <jsp:include page="actions/refer_form.jsp"/>
        </sec:authorize>
        <sec:authorize access="hasAuthority('USER')">
            <jsp:include page="actions/cancel_form.jsp"/>
        </sec:authorize>
```
than he/she should be allowed...or?

Reply:
because of this:     
```
<c:if test="${application.status == 'ALLOWED' || application.status == 'TEMPORARY_ALLOWED'}">
        <c:if test="${IS_OFFICE || (IS_USER && application.person.id == signedInUser.id)}">
            <jsp:include page="actions/cancel_form.jsp"/>
        </c:if>
    </c:if>
```

Reply:
> Office does not get any notification at all if person is in a department with departmentHead. [...]

That is what I wanted to achieve. If we have a responsible department head i dont think that the office has to know everything via mail. I think that the department members and the department head can do that by their own.

Reply:
>  <c:if test="${IS_OFFICE || (IS_USER && application.person.id == signedInUser.id)}">

oke...hm.